### PR TITLE
Switch between Artist Insights variations based on A/B enrollment

### PR DESCRIPTION
**Pull request by dblandin**
`v1` -> Artist Insights v1
`v2_before_bio` -> Artist Insights v2 before bio
`v2_after_bio` -> Artist Insights v2 after bio

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 4936652a5fb0cc3953653b9751415dfce7feff2f

https://deploy-preview-1852--artsy-reaction.netlify.com

**Comment by mzikherman** [comment]
This seems mergeable to me.

**Comment by dblandin** [comment]
@mzikherman I'm having some trouble with hydration after SSR.

Seems like the SSR isn't getting the experiment variation and is defaulting to ArtistInsightsV2. The client-side rehydration does seem to get the experiment variation, tries to render it and runs into some trouble with the diff patch.

@zephraph recommended trying to use the buildClientApp context, but I'm not sure of how to get the experiment variation value directly with the ArtistApp client.

**Comment by mzikherman** [comment]
Yea, I think this is an issue with the way we're accessing sharify on the server-side.

We're doing `require('sharify').data`, where as it _should_ be `res.locals.sd`, since this is a request-specific value. I don't think there's anything setup to pass those in via `buildServerApp`.

We can discuss with @damassi but we should be able to pass those in via context for the server (I believe what's happening here is the client-side rehydration is correct and it's the SSR version that is not seeing the proper sharify value). We already pass in certain things specifically from `res.locals.sd` here: https://github.com/artsy/force/blob/7391c26af635839d8561f0f23270f12d573906cc/src/desktop/lib/buildServerAppContext.ts#L15

File: src/Apps/Artist/Routes/Overview/index.tsx [hunk 1]
```diff
@@ -59,15 +62,56 @@ class OverviewRoute extends React.Component<OverviewRouteProps, State> {
...
+  }
+
   render() {
-    const { artist, user } = this.props
-    const showSelectedExhibitions = Boolean(artist.exhibition_highlights.length)
+    const { artist } = this.props
+    const artistInsightsVariation =
+      this.props.ARTIST_INSIGHTS || sd.ARTIST_INSIGHTS
+    const showArtistInsightsV1 =
+      (artistInsightsVariation === undefined ||
```
Comment (mzikherman) [inline src/Apps/Artist/Routes/Overview/index.tsx]: I don't think you need these `undefined` checks anymore.

**Pull request merged**